docs/sources/storage/_index.md

```diff
@@ -281,3 +285,33 @@ schema_config:
         prefix: index_
         period: 24h
 ```
+
+### Azure Storage Account
+
+```yaml
+schema_config:
+  configs:
+  - from: "2020-12-11"
+    index:
+      period: 24h
+      prefix: index_
+    object_store: azure
+    schema: v11
+    store: boltdb-shipper
+  storage_config:
+    azure:
+      # About how to get this value, see docs: https://docs.microsoft.com/en-us/azure/storage/common/storage-account-keys-manage?tabs=azure-portal
+      account_key: xxx
+      # Your azure account name
+      account_name: xxx
+      # See https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blobs-introduction#containers
+      container_name: xxx
```

Comment:
Please use a different substitution than xxx, as the other examples in this section have done.

```suggestion
    azure:
      # For the account-key, see docs: https://docs.microsoft.com/en-us/azure/storage/common/storage-account-keys-manage?tabs=azure-portal
      account_key: <account-key>
      # Your azure account name
      account_name: <account-name>
      # See https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blobs-introduction#containers
      container_name: <container-name>
```